test/api_key_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
 )
 
-func (s *CLITestSuite) TestAPIKey() {
+func (s *CLITestSuite) TestApiKey() {
```

Comment:
My other PR will do this renaming, so you can leave it off of this one. Same with the other two further down.